lib/cli/Cargo.toml

```diff
@@ -62,7 +63,7 @@ dirs = { version = "4.0" }
 serde_json = { version = "1.0" }
 target-lexicon = { version = "0.12", features = ["std"] }
 prettytable-rs = "0.9.0"
-wapm-toml = "0.2.0"
+wapm-toml = "0.4.0"
```

Comment:
Can we rename this to wasmer-toml ?

Reply:
We can after https://github.com/wasmerio/wapm-toml/pull/8 has been merged and we've published a new version.